command/v6/remove_network_policy_command_test.go

```diff
@@ -106,17 +138,82 @@ var _ = Describe("remove-network-policy Command", func() {
 			})
 		})
 
+		When("a destination space but no destination org is specified", func() {
+			BeforeEach(func() {
+				cmd.DestinationSpace = "dest-space"
+				fakeActor.GetSpaceByNameAndOrganizationReturns(v3action.Space{GUID: "some-dest-guid"}, v3action.Warnings{}, nil)
+			})
+
+			It("displays OK when no error occurs", func() {
+				Expect(executeErr).ToNot(HaveOccurred())
+				Expect(fakeActor.RemoveNetworkPolicyCallCount()).To(Equal(1))
+				Expect(fakeActor.GetOrganizationByNameCallCount()).To(Equal(0))
+				passedSrcSpaceGuid, passedSrcAppName, passedDestSpaceGuid, passedDestAppName, passedProtocol, passedStartPort, passedEndPort := fakeActor.RemoveNetworkPolicyArgsForCall(0)
+				Expect(passedSrcSpaceGuid).To(Equal("some-space-guid"))
+				Expect(passedSrcAppName).To(Equal("some-app"))
+				Expect(passedDestSpaceGuid).To(Equal("some-dest-guid"))
+				Expect(passedDestAppName).To(Equal("some-other-app"))
+				Expect(passedProtocol).To(Equal("tcp"))
+				Expect(passedStartPort).To(Equal(8080))
+				Expect(passedEndPort).To(Equal(8081))
+
+				Expect(testUI.Out).To(Say(`Removing network policy for app %s in org some-org / space some-space as some-user\.\.\.`, srcApp))
+				Expect(testUI.Out).To(Say("OK"))
+			})
+		})
+
+		When("a destination org is provided without a destination space", func() {
+			BeforeEach(func() {
+				cmd.DestinationOrg = "coolorg"
+			})
+
+			It("responds with an error", func() {
+				Expect(executeErr).To(MatchError(translatableerror.NetworkPolicyDestinationOrgWithoutSpaceError{}))
+			})
+		})
+
+		When("an invalid org is provided", func() {
+			BeforeEach(func() {
+				cmd.DestinationOrg = "coolorg"
+				cmd.DestinationSpace = "coolspace"
+				fakeActor.GetOrganizationByNameReturns(v3action.Organization{}, v3action.Warnings{}, actionerror.OrganizationNotFoundError{Name: "coolorg"})
```

Comment:
Please add warnings here and corresponding assertions testing that those warnings are printed.

Reply:
done